Makefile.am

```diff
@@ -137,71 +141,147 @@ EXTRA_DIST = 	\
 	autogen.sh \
 	README \
 	FAQ \
+	README.md \
 	README.win32 \
 	README.MacOSX \
 	Makefile.common \
 	Makefile.mingw \
-	Makefile.studio.mingw \
 	Info.plist.in \
-	exult.spec \
-	windrag.cc	\
+	stamp-h.in \
+	define-regex.txt \
```

Comment:
Hm. I committed this file by mistake; I will remove it from git, and you can remove it from this file.